tests/nnstreamer_rate/runTest.sh

```diff
@@ -31,28 +31,35 @@ callCompareTest original_0.log result_0.log 2-1 "Compare 2-1" 1 0
 callCompareTest original_150.log result_75.log 2-2 "Compare 2-2" 1 0
 callCompareTest original_200.log result_100.log 2-3 "Compare 2-3" 1 0
 
+rm result_*.log
+
 # Adjust frame rates (30 --> 3), downscaled with frame drops
 gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=300 ! video/x-raw,width=640,height=480,framerate=30/1 ! tensor_converter ! tensor_rate framerate=3/1 throttle=false ! multifilesink location=result_%1d.log" 3 0 0 $PERFORMANCE
 
 callCompareTest original_0.log result_0.log 3-1 "Compare 3-1" 1 0
 callCompareTest original_150.log result_15.log 3-2 "Compare 3-2" 1 0
 callCompareTest original_200.log result_20.log 3-3 "Compare 3-3" 1 0
 
+rm result_*.log
+
 # Adjust frame rates (30 --> 45), upscaled with frame duplication
 gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=300 ! video/x-raw,width=640,height=480,framerate=30/1 ! tensor_converter ! tensor_rate framerate=45/1 throttle=false ! multifilesink location=result_%1d.log" 4 0 0 $PERFORMANCE
 
 callCompareTest original_0.log result_0.log 4-1 "Compare 4-1" 1 0
 callCompareTest original_150.log result_225.log 4-2 "Compare 4-2" 1 0
 callCompareTest original_200.log result_300.log 4-3 "Compare 4-3" 1 0
 
+rm result_*.log
+
 # Adjust frame rates (30 --> 60), upscaled with frame duplication
 gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=300 ! video/x-raw,width=640,height=480,framerate=30/1 ! tensor_converter ! tensor_rate framerate=60/1 throttle=false ! multifilesink location=result_%1d.log" 5 0 0 $PERFORMANCE
 
 callCompareTest original_0.log result_0.log 5-1 "Compare 5-1" 1 0
 callCompareTest original_150.log result_300.log 5-2 "Compare 5-2" 1 0
 callCompareTest original_200.log result_400.log 5-3 "Compare 5-3" 1 0
 
-rm original_*.log
 rm result_*.log
 
+rm original_*.log
```

Comment:
Q) why do you move this line?

Reply:
Oh I think I did it for consistency of `rm result_*.log` after `callCompareTest`